cli_reference/openshift_developer_cli/using_odo_in_a_restricted_environment/creating-and-deploying-a-component-to-the-disconnected-cluster.adoc

```diff
@@ -0,0 +1,18 @@
+[id="creating-and-deploying-a-component-to-the-disconnected-cluster"]
+= Creating and deploying a component to the disconnected cluster
+include::modules/openshift-developer-cli-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: creating-and-deploying-a-component-to-the-disconnected-cluster
+toc::[]
+
```

Comment:
I really do think that this restructure is an improvement, breaking the assemblies out. However, now this assembly (and most of its procedures) don't have any lead-in information. Suggest adding at least at the top of the assembly, some information on what they're doing here/why.